selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml

```diff
@@ -69,25 +69,15 @@
             <class name="org.eclipse.che.selenium.debugger.DebugExternalClassTest">
                 <methods>
                     <exclude name="shouldDebugJreClass"/>
-                    <exclude name="shouldDebugMavenArtifactClassWithSources"/>
-                    <exclude name="shouldHandleDebugOfMavenArtifactWithoutSources"/>
                 </methods>
             </class>
             <class name="org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest">
                 <methods>
-                    <exclude name="shouldDebugInnerClass"/>
-                    <exclude name="shouldDebugLambdaExpressions"/>
                     <exclude name="shouldDebugMethodLocalInnerClass"/>
-                    <exclude name="shouldDebugStaticInnerClass"/>
                 </methods>
             </class>
             <class name="org.eclipse.che.selenium.debugger.MultimoduleProjectDebuggingTest"/>
-            <class name="org.eclipse.che.selenium.debugger.NodeJsDebugTest">
-                <methods>
-                    <exclude name="debugNodeJsTest"/>
```

Comment:
*debugNodeJsTest* was excluded from the stable test list because of it sometimes fail: https://ci.codenvycorp.com/view/qa/job/che-integration-tests/55/testReport/org.eclipse.che.selenium.debugger/NodeJsDebugTest/